index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Portifolio</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="styles.css">
+  </head>
+
+  <body>
+
+      <div class="toolbar">
+        <img class="nav-img" src="C:\Users\Ivan\Desktop\Setup-and-mobile-first\images\Toolbar - Dark - Compact.png">
+        <button class="btn"><i class="bars"></i></button>
+      </div>
+
+    <img class="back-img" src="images\header-shapes mobile.png">
+
+    <div class="intro">
+      <h1 class="greeting">I'm Ivan<br>Glad to meet <br>you!</h1>
+      <p class="descript">I'm a software developer! I can help you build a product, feature or website look at some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+    </div>
+
+    <div class="socials-div">
+      <a class="socials" href="">LET'S CONNECT</a><br>
+      <button class="socials">
+        <div class="social-media">
+          <ul>
+            <li class="linkedin"><a href="https://github.com/IvanLudick"><img src="images\Linkedin icon.jpg"></a></li>
+            <li class="gmail"><img src="images\Gmail.jpg"></li>
+            <li class="github"><img src="images\GitHub.jpg"></li>
+            <li class="twitter"><img src="images\Twitter.jpg"></li>
+          </ul>
```

Comment:
[OPTIONAL] Great job adding the `ul` wrapping your `li` elements. Please keep in mind that these icons should redirect the user to your social media 😄. To do so you can create `a` tags containing the link to your profile, and add the icon as an image.

Here is my tip for you :wink:: 
```html 
<ul> 
  <li><a href="your social media URL">the social media icon</a></li> 
  <li><a href="another social media URL">the social media icon</a></li> 
  <li><a href="another social media URL">the social media icon</a></li> 
</ul>
```

More information can be found [here](https://www.w3schools.com/tags/tag_a.asp).